crates/bindings-csharp/BSATN.Runtime/Builtins.cs

```diff
@@ -61,67 +70,69 @@ string wrapperPropertyName
         );
 }
 
-public record Address : BytesWrapper
+public readonly record struct Address
 {
-    protected override int SIZE => 16;
+    private readonly U128 value;
 
     public Address() { }
```

Comment:
These default constructors here and in `Identity` can be removed now; they were necessary for `record` but any `struct` has them automatically.